include/GL/glui.h

```diff
@@ -581,7 +642,7 @@ class GLUIAPI GLUI_Glut_Window : public GLUI_Node
     void (*glut_mouse_CB)(int,int,int,int);
     void (*glut_visibility_CB)(int);
     void (*glut_motion_CB)(int,int);
-    void (*glut_display_CB)(void);
```

Comment:
I opened a seperate PR for [void](https://github.com/libglui/glui/pull/65).
[Travis build](https://travis-ci.org/nigels-com/glui/builds/245659828) is green.
Feedback appreciated.